### ISSUE TITLE
Training new models

### ISSUE THREAD

**Issue by rodrigo-puente** [google/emoji-scavenger-hunt#38]
I followed the instructions to build my own model. Added 190 images of smiles and 190 images of eyes, all cropped, same size (200x100). 

I ran the docker command with the default settings, 4000 steps. Here is part of the console output:

`INFO:tensorflow:2018-09-11 23:16:32.141023: Step 3100: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:16:32.141567: Step 3100: Cross entropy = 0.000412
INFO:tensorflow:2018-09-11 23:16:32.537921: Step 3100: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:17:17.138179: Step 3200: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:17:17.138496: Step 3200: Cross entropy = 0.000361
INFO:tensorflow:2018-09-11 23:17:17.527896: Step 3200: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:17:56.203106: Step 3300: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:17:56.203532: Step 3300: Cross entropy = 0.000178
INFO:tensorflow:2018-09-11 23:17:56.570616: Step 3300: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:18:40.495192: Step 3400: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:18:40.495422: Step 3400: Cross entropy = 0.000145
INFO:tensorflow:2018-09-11 23:18:40.999290: Step 3400: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:19:34.927443: Step 3500: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:19:34.927855: Step 3500: Cross entropy = 0.000417
INFO:tensorflow:2018-09-11 23:19:35.369817: Step 3500: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:20:21.830019: Step 3600: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:20:21.830568: Step 3600: Cross entropy = 0.000333
INFO:tensorflow:2018-09-11 23:20:22.357888: Step 3600: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:21:09.905047: Step 3700: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:21:09.905420: Step 3700: Cross entropy = 0.000110
INFO:tensorflow:2018-09-11 23:21:10.347650: Step 3700: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:21:57.355424: Step 3800: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:21:57.355648: Step 3800: Cross entropy = 0.000422
INFO:tensorflow:2018-09-11 23:21:57.912983: Step 3800: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:22:43.254789: Step 3900: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:22:43.255021: Step 3900: Cross entropy = 0.000189
INFO:tensorflow:2018-09-11 23:22:43.670463: Step 3900: Validation accuracy = 100.0% (N=100)
INFO:tensorflow:2018-09-11 23:23:24.426763: Step 3999: Train accuracy = 100.0%
INFO:tensorflow:2018-09-11 23:23:24.427008: Step 3999: Cross entropy = 0.000133
INFO:tensorflow:2018-09-11 23:23:24.857894: Step 3999: Validation accuracy = 100.0% (N=100)
`

I added all the files, modified everything and got it working, but no matter what I put in front of the camera, it turns out to be correct everytime for eyes and smiles. 

Would you suggest more images? More steps? Bigger images?

**Comment by tushuhei** [comment]
Hi there,
Thank you for using the game! The default game logic captures top 2 objects as found items to relax the severeness.
https://github.com/google/emoji-scavenger-hunt/blob/49aa1103b0f0f22a2f4e7d69c7246cc8971dddd4/src/js/game.ts#L386

If you want to use just top 1 object, you may change this line to like:
```
this.checkEmojiMatch(topK[0].label, 'dummy');
```

**Comment by tushuhei** [comment]
Let me close this now. Feel free to re-open if you’re still seeing the issue.
